src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+
+import MainContent from './main-content/MainContent';
+import SecondaryContent from './secondary-content/SecondaryContent';
+import { ImageButton } from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as chevronLeftIcon } from 'static/img/shared/chevron-left.svg';
+import { ReactComponent as chevronRightIcon } from 'static/img/shared/chevron-right.svg';
+
+import { Status } from 'src/shared/types/status';
+
+import styles from './TranscriptsTable.scss';
+
+const TranscriptsTable = () => {
+  const [paginationNumber, setPaginationNumber] = useState(1);
+
+  const gotoPrevious = () => {
+    if (paginationNumber > 1) {
```

Comment:
Should the button even be clickable if this condition is not satisfied? If not, this should probably be done at the point of the styling the button / adding click handler.

Alternatively, is it possible that the table will show the last page if user clicks on the button when page is 1? What does Andrea say?

Reply:
It should keep rotating the panel e.g. when at panel 3 and next is clicked -> show panel 1

Reply:
:heavy_check_mark: